app/models/observation.rb

```diff
@@ -46,7 +46,7 @@ class Observation < ApplicationRecord
   belongs_to :programme, optional: true
   belongs_to :audit, optional: true
   belongs_to :school_target, optional: true
-  enum observation_type: [:temperature, :intervention, :activity, :event, :audit, :school_target, :programme]
+  enum observation_type: [:temperature, :intervention, :activity, :event, :audit, :school_target, :programme, :audit_activities_completed]
```

Comment:
Adding a new Observation type means you also need to add a timeline ERB file so that the observation displays as part of the school timeline, as you did for `:programme` in a recent PR. Without that when the observation is added the school timeline will break. So you'll need to add a template for that.

Reply:
:+1: done (and a reminder comment added too)